builders/ethereum/dev-env/thirdweb.md

```diff
@@ -167,75 +181,75 @@ function App() {
 
 For extension based functions, use the built-in supported hooks. There are several hooks available for you to use, the following are a few examples:
 
-- Use the NFTs extension to access a list of NFTs owned by an address via the [`useOwnedNFTs` hook](https://portal.thirdweb.com/react/react.useownednfts){target=\_blank}:
+- Use the NFTs extension to access a list of NFTs owned by an address via the [`useOwnedNFTs` hook](https://portal.thirdweb.com/references/react/v4/useOwnedNFTs){target=\_blank}:
 
-    ```javascript
-    import { useOwnedNFTs, useContract, useAddress } from '@thirdweb-dev/react';
+```javascript title="Call Contract Functions"
+import { useOwnedNFTs, useContract, useAddress } from '@thirdweb-dev/react';
 
-    // Your smart contract address
-    const contractAddress = 'INSERT_CONTRACT_ADDRESS';
+// Your smart contract address
+const contractAddress = 'INSERT_CONTRACT_ADDRESS';
 
-    function App() {
-      const address = useAddress();
-      const { contract } = useContract(contractAddress);
-      const { data, isLoading, error } = useOwnedNFTs(contract, address);
-    }
-    ```
+function App() {
+  const address = useAddress();
+  const { contract } = useContract(contractAddress);
+  const { data, isLoading, error } = useOwnedNFTs(contract, address);
+}
+```
 
-- Use the [`useContractRead` hook](https://portal.thirdweb.com/react/react.usecontractread){target=\_blank} to call any read functions on your contract by passing in the name of the function you want to use:
+- Use the [`useContractRead` hook](https://portal.thirdweb.com/references/react/v4/useContractRead){target=\_blank} to call any read functions on your contract by passing in the name of the function you want to use:
 
-    ```javascript
-    import { useContractRead, useContract } from '@thirdweb-dev/react';
+```javascript title="Read Contract Data"
+import { useContractRead, useContract } from '@thirdweb-dev/react';
```

Comment:
have these changed with v5? and how are we supposed to access these now from @thirdweb/react instead of @thirdweb-dev/react?